tox.ini

```diff
@@ -7,17 +7,9 @@ envlist =
 requires =
     setuptools >= 30.3.0
     pip >= 19.3.1
-    tox-pypi-filter >= 0.12
-isolated_build = true
```

Comment:
So we don't need `isolated_build = true` because it is always true now, is that correct?

Reply:
Yes, the option got removed